notebooks/ullyses-walkthrough.ipynb

```diff
@@ -302,7 +373,17 @@
    "outputs": [],
    "source": [
     "# Print the columns in the science extension of the HDU - the science extension is index 1\n",
-    "print(spec_hdu[1].columns)"
+    "spec_filename = glob.glob(f'*xx-cha*_preview-spec.fits')[0]\n",
+    "print(spec_filename)\n",
+    "with fits.open(spec_filename) as spec_hdu:\n",
+    "    print(spec_hdu[1].columns)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's plot the flux, error, wavelength, and SNR for each level 4 (`preview-spec`) dataset. To get these values we use \".data\" to access the data in the science extension of the HDU. To access the header of this extension instead, we would use \".header\""
```

Comment:
Suggest putting .data and .header in code block quotes like `.data` and `.header`